### DISCUSSION TITLE
Examples Built with VScode for WIFI for Pico W will not connect, but work with the Arduino IDE.  How can I make VScode work?

### DISCUSSION THREAD

**Discussion by frohro** [earlephilhower/arduino-pico#2051] [Q&A] [answered]
So ,  I like using Github Copilot and VScode, but when I take the WIFI example for UDP or WIFI Client and build them on VScode using platformio, they build fine, but they will not connect to my WIFI access point, but they work fine when compiled with the Arduino IDE.  I have been just copy/pasting back to the Arduino IDE to build, and flash, but keeping everything in sync is a nightmare.  :-)  It also too me a long time to figure this out.  It would be nice to fix the problem.

**Comment by earlephilhower** [comment, marked as answer]
You need to set the board to `rpipicow` in order to get the WiFi blob included.  Double check your `ini` file.
